### PR TITLE
Fixed the CreateWriteMessage for the PN532 chip.

### LINKED ISSUE #2075: The PN532 message factory produces bugged extended frames
**Describe the bug**

The PN532 message factory produces extended frames that are not compliant with 1.6 firmware specifications as reported at page 29 of the official manual https://www.nxp.com/docs/en/user-guide/141520.pdf
After sending a bugged  extended frame there is no response.

**Steps to reproduce**
Checkout 02437dfa295cf8b627fb734236a79c16881d0cb7
Force the use of extended frames replacing line 1571 of Pn532.cs
```csharp
            int correctionLArgeSizeBuffer = writeData.Length < 250 ? 0 : 2;
```
with 
```csharp
            int correctionLArgeSizeBuffer = 2;
```
Run mifare test of the Pn532sample to verify that is broken.
Tested the sample on pn532 connected via serial port.

**Expected behavior**

The extended frame should work as the normal frame does, as reported in the manual at page 29.

**Actual behavior**

Every command does not receive a response in the default time (89 ms).

**Versions used**
.NET SDK:
 Version:   7.0.203
 Commit:    5b005c19f5

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19045
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\7.0.203\

Host:
  Version:      7.0.5
  Architecture: x64
  Commit:       8042d61b17

.NET SDKs installed:
  5.0.416 [C:\Program Files\dotnet\sdk]
  7.0.203 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.15 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.15 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.5 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

- Version of `System.Device.Gpio`: dont know, is not released yet
- Version of `Iot.Device.Bindings` package: dont know, is not released yet

### PR DESCRIPTION
**Pull request by pianca**
Fixes #2075 
Fixed the PN532 message factory that now produce extended frames compliant with 1.6 firmware specifications

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/2076)

### CODE REVIEW

**Comment by pianca** [comment]
@dotnet-policy-service agree

**Review by Ellerbach — APPROVED** [review]
Thanks a lot for fixing this and testing it!
Looks good to me!

File: src/devices/Pn532/Pn532.cs [hunk 1]
```diff
@@ -18,6 +18,8 @@
 using Iot.Device.Rfid;
 using Iot.Device.Pn532;
 using Microsoft.Extensions.Logging;
+using SixLabors.ImageSharp.PixelFormats;
```
Comment (pgrawehr) [inline src/devices/Pn532/Pn532.cs:21]: This using statement seems to be in error.

File: src/devices/Pn532/Pn532.cs [hunk 2]
```diff
@@ -18,6 +18,8 @@
...
+using UnitsNet;
```
Comment (Ellerbach) [inline src/devices/Pn532/Pn532.cs:22]: And while we're there, this one does not seems to be needed neither!

File: src/devices/Pn532/Pn532.cs (hunk 1 above)
Comment (Ellerbach) [reply src/devices/Pn532/Pn532.cs:21]:
```suggestion
```

File: src/devices/Pn532/Pn532.cs (hunk 2 above)
Comment (Ellerbach) [reply src/devices/Pn532/Pn532.cs:22]:
```suggestion
```

**Comment by pianca** [comment]
I removed those using (don't know why they got added) and rand the build.net script successfully

**Review by pgrawehr — APPROVED** [review]

**Pull request merged**